stdlib/ballerina-grpc/src/main/java/org/ballerinalang/net/grpc/ssl/SSLHandlerFactory.java

```diff
@@ -0,0 +1,188 @@
+/*
+ *  Copyright (c) 2015 WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Same as above

Reply:
Fixed in b83bfde0a841ea11c5fe83d8eeca9a2d5fd6764c